src/elements/hx-switch/index.spec.js

```diff
@@ -46,6 +46,16 @@ describe('<hx-switch> component tests', () => {
 
             expect(attr).to.be.true;
         });
+
+        it('should have toggled attribute', async () => {
+            const fragment = /** @type {HXToggleElement} */ await fixture(mockup);
+            const attr = fragment.hasAttribute('toggled');
+            fragment.toggled = true;
+            const checkattr = fragment.hasAttribute('toggled');
```

Comment:
Please change variable to `toggledAttr`.

Reply:
Changes have been done, please review.